tools/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -414,67 +441,91 @@ public partial class OutlineService : ServiceBase {
       //    client's call to ResetRouting).
       var interfacesWithIpv4Gateways = NetworkInterface.GetAllNetworkInterfaces()
           .Where(i => i.Name != TAP_DEVICE_NAME)
+          .Where(i => i.OperationalStatus == OperationalStatus.Up)

```

Comment:
Let's add a note explaining that we've seen reports where this function failed because inactive interfaces had a gateway.

Reply:
Done.